drupal-check/drupal-check

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+VERSION="1.0"
+
+## Check Drupal code for deprecations and discover bugs via static analysis.
+##
+## Built on PHPStan, this static analysis tool will check
+## for correctness, deprecation errors, and more
+##
+##  fin drupal-check <module_name> <options>
+##
+
+## Options
+##    -a        Check analysis
+##    -d        Check deprecations (default)
+##    -t        Module type {core, custom, profile, contrib (default)}
+##    -v        Verbose output.
+
+# Default variables.
+SCRIPTS_DIR="/var/www/.docksal/addons/drupal-check/scripts"
+MODULE_TYPE="contrib"
+CHECK_OPTS=()
+PROFILE_DIR=""
+SCRIPT_ARG=""
+
+# Import local settings
+[[ -f local.drupal-check ]] && source local.drupal-check || CUSTOM=false
```

Comment:
Is it by design that the file should always be in a current directory?

Reply:
This is a check to see if local settings are there. We've included an `example.local.drupal-check` file as an example.